addons/web/static/src/legacy/js/libs/bootstrap.js

```diff
@@ -160,6 +160,22 @@ Dropdown.prototype._detectNavbar = function () {
     return this._element.closest(".navbar-collapse.show");
 };
 
+/**
+ * As explained above, the dynamic navbar's dropdown positioning was activated
+ * to prevent sub-menus overflow. This positioning will use the default BS
+ * offsets to position sub-menus leading to a small gap that hides them when
+ * hovered (on "Hover" mode). The goal here is to prevent this offset when the
+ * target is inside a navbar.
+ */
+const bsGetOffsetFunction = Dropdown.prototype._getOffset;
+Dropdown.prototype._getOffset = function () {
+    const offset = bsGetOffsetFunction.apply(this, arguments);
+    if (this._element.closest(".navbar")) {
```

Comment:
```suggestion
    if (this._element.closest(".o_hoverable_dropdown .navbar")) {
```
I would only apply this diff when the "On Hover" option is enabled, since the default (and only) BS behavior is on click which does not have any issue.

Reply:
Yes, it's definitely better this way. Code & commit message updated. Thanks :+1: